### ISSUE TITLE
Display link to action run

### ISSUE THREAD

**Issue by kevinsmith** [daun/playwright-report-summary#205]
Is it possible to get the PR comment to link to the Action run itself (like this: https://github.com/daun/playwright-report-summary/actions/runs/9828037864), so the reader can dig in for more details?

**Comment by daun** [comment]
You can use the `report-url` input to make it render a `Open report` link to that specific url. A quick google says there's a `run_id` context variable, which would yield this (untested) example. Does that work for you?

```yaml
- uses: daun/playwright-report-summary@v3
  with:
    # URL to a report, uploaded or generated in a previous step
    report-url: 'https://github.com/org/repo/actions/runs/${{ github.run_id }}'
```

**Comment by daun** [comment]
If you need control over the link label, you can pass in your own markdown info through the `custom-info` input:

```yaml
- uses: daun/playwright-report-summary@v3
  with:
    custom-info: '[Link label](link)'
```

**Comment by kevinsmith** [comment]
That worked perfectly, thanks!

**Comment by kevinsmith** [comment]
Oh heads up, I'm now getting this under annotations on the workflow run summary page.

<img width="1227" alt="image" src="https://github.com/daun/playwright-report-summary/assets/397904/7774fb10-1f0b-4fac-8e12-35182f0dab8f">

**Comment by daun** [comment]
Nevermind, this was due to a missing input parameter in the action manifest. Fixed in 3.5.2.
